src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.Api/Controllers/ApprenticeFeedbackEmailTransactionController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.ApprenticeFeedback.Application.Commands.GenerateEmailTransaction;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeFeedback.Api.Controllers
+{
+    [ApiController]
+    [Route("FeedbackTransaction")]
```

Comment:
Usually we default to:
```    [Route("[controller]/")]
    public class ApprenticeFeedbackTargetController : Controller```
    
    Where the controller is the name of the endpoint, can we do the same here to keep it consistent with the other controllers?

Reply:
Changed routing as suggested